hooks.php

```diff
@@ -346,11 +346,8 @@
 // Sentry initializer - Only for staging and production environments
 // -------------------------------------------------------------------------------------------------------------------
 if (
-	! is_null( env( 'WP_ENV' ) ) &&
-	WP_ENV !== 'development' &&
-	! is_null( env( 'SENTRY_KEY' ) ) &&
-	! is_null( env( 'SENTRY_ORGANIZATION' ) ) &&
-	! is_null( env( 'SENTRY_PROJECT' ) )
+	! is_null( env( 'ENABLE_SENTRY' ) ) &&
```

Comment:
Concern for backwards compatibility since environment variables are removed and new ones are added.

Reply:
It is compatible, since other variables (the previous ones) are not longer used, then are not considered in any part of the code.